refgenconf/refgenconf.py

```diff
@@ -90,17 +94,22 @@ def _missing_key_msg(key, value):
             try:
                 version = float(version)
             except ValueError:
-                _LOGGER.warning("Cannot parse config version as numeric: {}".format(version))
+                _LOGGER.warning(
+                    "Cannot parse config version as numeric: {}".format(version))
             else:
                 if version < REQ_CFG_VERSION:
                     msg = \
-                        "This genome config (v{}) is not compliant with v{} standards. " \
-                        "To use it, please downgrade refgenie: 'pip install \"refgenie>={},<{}\"'".\
-                            format(self[CFG_VERSION_KEY], str(REQ_CFG_VERSION),
-                                   REFGENIE_BY_CFG[str(version)], REFGENIE_BY_CFG[str(REQ_CFG_VERSION)])
+                        "This genome config (v{}) is not compliant with v{} standards. \n" \
+                        "To use it, please upgrade the config file: 'refgenie upgrade --target-version {}', or " \
+                        "downgrade refgenconf: 'pip install \"refgenconf>={},<{}\"'".\
```

Comment:
downgrading just the `refgenconf` package will not make `refgenie` work with the previous version since the latest `refgenie` will still expect the reformatted config.

So I think in this message we need to explicitly mention both packages. Also, since `config_upgrade` is a function in the public `refgenconf` API, let's mention this as another way to upgrade.

Reply:
> downgrading just the `refgenconf` package will not make `refgenie` work with the previous version since the latest `refgenie` will still expect the reformatted config.

how to get the correct `refgenie` version that the compatible with v0.3? As Dr. Sheffield mentioned in https://github.com/refgenie/refgenie/issues/205 the current version refgenie is already > `REFGENIE_BY_CFG[str(version)]`. 

> Also, since `config_upgrade` is a function in the public `refgenconf` API, let's mention this as another way to upgrade.

do you mean change this `please upgrade the config file: 'refgenie upgrade --target-version {}'` to something else? or change the way to upgrade?

Reply:
> how to get the correct refgenie version that the compatible with v0.3? 

I don't think there's a way unless we put this info into the package.. And that's the point, we should make this message independent of `refgenie` since it's in `refgenconf`.

> do you mean change this please upgrade the config file: 'refgenie upgrade --target-version {}' to something else? or change the way to upgrade?

Indicate `config_upgrade` as the way to upgrade and maybe add "or use `refgenie upgrade` to do this from the CLI`. But this is just an alternative way to do this since we don't know if the user even has `refgenie` package installed.

Reply:
> we should make this message independent of `refgenie` since it's in `refgenconf`.

How about make the message like: "This genome config (v{}) is not compliant with v{} standards. To use current refgenconf, please use config_upgrade(target_version, file_path, force) to upgrade, or downgrade refgenconf: pip install refgenconf>={},<{}. I you are using refgenie, please use 'refgenie upgrade --target-version {}' to upgrade, or downgrade refgenie: 'pip install refgenie>={},<{}."


So `REFGENIE_BY_CFG` should be` REFGENCONF_BY_CFG` since v0.3 works for 0.9.0 `refgenie`
```REFGENIE_BY_CFG = {"0.4": "0.10.0", "0.3": "0.7.0", "0.2": "0.6.0"}``` 
then, what is the correct config version:refgenie version?

Reply:
> what is the correct config version:refgenie version

let's not bother with config version:refgenie version relationship. Just remove that from the error message:

"This genome config (v{}) is not compliant with v{} standards. To use current refgenconf, please use config_upgrade function to upgrade, or downgrade refgenconf: pip install refgenconf>={},<{}. If refgenie is installed, you can use 'refgenie upgrade --target-version {}'"

Reply:
ok